### PR TITLE
ddl: Add the function of close

### PR DESCRIPTION
**Pull request by zimulala**
- add logs for starting or stopping DDL

- add the function of close after use. Otherwise，this DDL want to be `owner`, it can interfere with the other tests.

- use Stop instead of close, make owner change faster.

### CODE REVIEW

**Comment by shenli** [comment]
LGTM

File: ddl/ddl.go [hunk 1]
```diff
@@ -1,4 +1,5 @@
 // Copyright 2013 The ql Authors. All rights reserved.
+
```
Comment (hanfei1991) [inline ddl/ddl.go]: Why add this ?

**Comment by shenli** [comment]
@hanfei1991 PTAL

**Comment by zimulala** [comment]
PTAL @hanfei1991

**Comment by hanfei1991** [comment]
LGTM

**Pull request merged**